mongo_orchestration/common.py

```diff
@@ -48,6 +49,7 @@ class BaseModel(object):
         {'role': 'dbAdminAnyDatabase', 'db': 'admin'},
         {'role': 'readWriteAnyDatabase', 'db': 'admin'}
     ]
+    socket_timeout = DEFAULT_SOCKET_TIMEOUT
```

Comment:
I don't really like this. Anywhere you call BaseModel.socket_timeout = <value> is going to change the value for all instances of subclasses. It's really inflexible.